### PR TITLE
Feature translation

### PR DESCRIPTION
**Pull request by lingy1028**
We raised this PR to close the below issues
- #9 - Format Trap Event to KV
- #10 - OID to MIB Translation
- #11 - OIB to Custom String Translation
- Changed default logging level from debug to info
- Fixed typo from the previous PR in `pyproject.toml` config file
- Replaced the real token with a placeholder in `config.yaml` config file

### CODE REVIEW

**Comment by mayurah** [comment]
- #21 - Dockerfile corrections

File: splunk_connect_for_snmp_traps/manager/trap_server.py [hunk 1]
```diff
@@ -19,73 +27,101 @@ def __init__(self, server_config):
...
+            "* contextEngineId: %s" % variables["contextEngineId"].prettyPrint()
+        )
+        logger.debug("* contextName: %s" % variables["contextName"].prettyPrint())
+        logger.debug("* PDU: %s" % variables["pdu"].prettyPrint())
 
     # Callback function for receiving notifications
     # noinspection PyUnusedLocal,PyUnusedLocal,PyUnusedLocal
-    def snmp_callback_function(self, snmp_engine, state_reference, context_engine_id, context_name, var_binds,
-                               callback_ctx):
+    def snmp_callback_function(
```
Comment (xynazog) [inline splunk_connect_for_snmp_traps/manager/trap_server.py:108]: Sometimes I like the old formatting. I like double quotes but this multiline function definition looks far from clean sometimes.

**Review by xynazog — APPROVED** [review]
Looks good to me.

**Review by mayurah — APPROVED** [review]
Looks good, thanks Yuan!

We can change the python linter in VSCode to [black](https://dev.to/adamlombard/how-to-use-the-black-python-code-formatter-in-vscode-3lo0)

File: splunk_connect_for_snmp_traps/manager/trap_server.py (hunk 1 above)
Comment (mayurah) [reply splunk_connect_for_snmp_traps/manager/trap_server.py:108]: Affirmative.

**Review by xynazog — APPROVED** [review]

**Pull request merged**